interfaces/Matlab_Toolbox_Revamp/Class/FlowDevice.m

```diff
@@ -0,0 +1,154 @@
+classdef FlowDevice < handle
+    
+    properties
+        type
+        id
+        upstream
+        downstream
+    end
+    
+    methods
+        %% FlowDevice class constructor
+
+        function x = FlowDevice(typ)
+            % Flow Device class constructor.
+            %
+            % parameter typ:
+            %    Type of flow device to be created. Type =
+            %    'MassFlowController', 'PressureController' or 'Valve'.
+            
+            checklib;
+            
+            if nargin == 0
+                typ = 'MassFlowController';
+            end
+            
+            if isa(typ, 'double')
+                warning(['Definition via integer type to be deprecated', ...
+                         ' after Cantera 2.5.']);
```

Comment:
This is going to get merged after 2.5, and the whole interface has a different... well, interface. So you don't need to add any deprecation warnings. You can just not implement this means of creating the class.